Source/Plugins/Core/com.equella.core/js/tsrc/mainui/Template.tsx

```diff
@@ -549,25 +575,39 @@ export const Template = React.memo(function Template({
     );
   }
 
-  const layout = useFullscreen({ fullscreenMode, hideAppBar }) ? (
-    <main>{children}</main>
-  ) : (
-    <div className={classes.appFrame}>
-      <AppBar className={classes.appBar}>
-        <Toolbar disableGutters>
-          {hasMenu && (
-            <IconButton
-              className={classes.navIconHide}
-              onClick={(_) => setNavMenuOpen(!navMenuOpen)}
-            >
-              <MenuIcon />
-            </IconButton>
-          )}
-          {titleArea()}
-          {menuArea()}
-        </Toolbar>
-        {tabs}
-      </AppBar>
+  function renderError(error: ErrorResponse) {
```

Comment:
```suggestion
  function RenderError({error}: {error: ErrorResponse}) {
```

would allow

```jsx
{errorResponse && renderError(errorResponse)}
```
to become
```jsx
{errorResponse && <RenderError error={errorResponse} />}
```

below

Reply:
Good plan. Actioned in commit ddc3a9769b2911ec53946999a4aac20c56343779